pkgs/development/python-modules/pyrevolve/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, contexttimer
+, versioneer
+, cython
+, numpy
+}:
+
+buildPythonPackage rec {
+  pname = "pyrevolve";
+  version = "2.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "b49aea5cd6c520ac5fcd1d25fa23fe2c5502741d2965f3eee10be067e7b0efb4";
+  };
+
+  buildInputs = [ versioneer cython numpy ];
+  propagatedBuildInputs = [ contexttimer ];
+
+  meta = with lib; {
+    homepage = "https://github.com/devitocodes/pyrevolve";
+    description = "Python library to manage checkpointing for adjoints";
+    license = licenses.epl10;
```

Comment:
Missing maintainer

Reply:
Hello Sandro, thx for the review yet again! Yeah, I need to fix that, I'll do it as soon as I can.